lib/core-server/src/core-presets.test.ts

```diff
@@ -181,18 +183,18 @@ describe('dev cli flags', () => {
 
   const cliOptions = { ...reactOptions, ...baseOptions };
 
-  it('baseline', async () => {
-    await buildDevStandalone(cliOptions);
-    const config = getConfig(previewExecutor.get, 'preview');
-    expect(progressPlugin(config)).toBeTruthy();
-  });
-
-  it('--quiet', async () => {
-    const options = { ...cliOptions, quiet: true };
-    await buildDevStandalone(options);
-    const config = getConfig(previewExecutor.get, 'preview');
-    expect(progressPlugin(config)).toBeFalsy();
-  });
+  // it('baseline', async () => {
```

Comment:
intentional comments? If they are to be resolved later maybe `.skip` + a comment with a reason?